src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,26 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: widthOfCalendar();
+  padding: 20px $paddingOfCalendar;
+  margin: 0 auto;
+
+  &__cell {
+    @include sizeOfCell;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    transition: background-color 0.5s, transform 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      cursor: pointer;
```

Comment:
Maybe will be better use it for &__cell in general, not only for :hover effect?

Reply:
Yes, makes sense.